ShareX.UploadersLib/FileUploaders/Aboo.cs

```diff
@@ -0,0 +1,125 @@
+﻿#region License Information (GPL v3)
+
+/*
+    ShareX - A program that allows you to take screenshots and share any file type
+    Copyright (c) 2007-2018 ShareX Team
+
+    This program is free software; you can redistribute it and/or
+    modify it under the terms of the GNU General Public License
+    as published by the Free Software Foundation; either version 2
+    of the License, or (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program; if not, write to the Free Software
+    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+    Optionally you can also view the license at <http://www.gnu.org/licenses/>.
+*/
+
+#endregion License Information (GPL v3)
+
+using Newtonsoft.Json;
+using System.IO;
+
+namespace ShareX.UploadersLib.FileUploaders
+{
+    public class AbooFileUploaderService : FileUploaderService
+    {
+        public override FileDestination EnumValue { get; } = FileDestination.Aboo;
+
+        public override bool CheckConfig(UploadersConfig config)
+        {
+            return config.AbooAccountType == AccountType.Anonymous;
```

Comment:
Not sure why this field is necessary because this is not set anywhere and therefore serves no purpose, not to mention that the service does not appear to offer accounts (and that "full anonymity" is their goal).